package.json

```diff
@@ -8,8 +8,8 @@
     "build": "next build",
     "start": "next start",
     "check-types": "tsc",
-    "e2e:ui": "playwright test --ui",
-    "e2e": "playwright test",
+    "test": "playwright test",
+    "test-ui": "playwright test --ui",
```

Comment:
it's OK, but why do you want to change this?

Reply:
no good reason, I am used to run yarn test  so it's faster for me

Reply:
ok, we usually run these commands for CI/CD. Did you search other places where we are using it??